docs/OncoTree-Mapping-Tool.md

```diff
@@ -184,3 +188,26 @@ After filling in the `ONCOTREE_CODE` column with an OncoTree code for each sampl
 |P16|S16|65|MNM|
 |P17|S17|66|MYCF|
 |P18|S18|66|RBL|
+
+# Ontology to Ontology Mapping Tool
+
+The Ontology Mapping tool was developed to facilitate the mapping between different cancer classification systems. We currently allow the mappings between OncoTree, ICD-O, NCIt, UMLS and HemeOnc systems. The tool and the mapping file can be found [here](https://github.com/cBioPortal/oncotree/tree/master/scripts/ontology_to_ontology_mapping_tool)
```

Comment:
Users do not need to download the mapping file in order to run the tool; I would make that clear here with a short statement (The mapping file does not need to be downloaded to run the tool). I now realize that in order for the user to be able to use the tool, they will need to have an internet connection so that the script can load the mapping file from github. Maybe internet connections are now an assumed part of life ... so maybe we don't need to say anything about it. Hopefully the script will output a sensible message if a user tries to run it with no network access.

Reply:
Updated the README to include the statement @sheridancbio